toolbox/deps_update/main.go

```diff
@@ -77,15 +93,18 @@ func updateDeps(repo string, deps *[]u.Dependency, depChangeList *[]u.Dependency
 	args := ""
 	for _, updatedDep := range *depChangeList {
 		switch updatedDep.RepoName {
-		case "mixer":
+		case mixerRepo:
 			args += fmt.Sprintf("-x %s,%s ", *hub, updatedDep.LastStableSHA)
-		case "pilot":
-			istioctlURL := fmt.Sprintf(
-				"https://storage.googleapis.com/istio-artifacts/pilot/%s/artifacts/istioctl",
-				updatedDep.LastStableSHA)
-			args += fmt.Sprintf("-p %s,%s -i %s ", *hub, updatedDep.LastStableSHA, istioctlURL)
-		case "auth":
-			args += fmt.Sprintf("-c %s,%s ", *hub, updatedDep.LastStableSHA)
+		case pilotRepo:
+			istioctlURL := generateArtifactURL(pilotRepo, updatedDep.LastStableSHA, istioctlSuffix)
+			debianUrl := generateArtifactURL(pilotRepo, updatedDep.LastStableSHA, debianSuffix)
+			args += fmt.Sprintf("-p %s,%s -i %s -P %s", *hub, updatedDep.LastStableSHA, istioctlURL, debianUrl)
```

Comment:
Need a white space at the end of string so that when two sets of flags are added to `args` they are separated. Same thing applies to other cases in this switch statement.